### ISSUE TITLE
Replace Rupee in Sans versions with Devanagari Rupee version

### ISSUE THREAD

**Issue by mjabbink** [IBM/plex#345]
<img width="567" alt="Screen Shot 2020-10-08 at 9 27 37 PM" src="https://user-images.githubusercontent.com/32881239/95535080-1fa79280-09ad-11eb-8c61-275a5bc0524d.png">

**Comment by BoldMonday** [comment]
Good idea. Means that we need to create this glyph for the italics and Sans Condensed fonts, since Devanagari only consists of upright styles.

**Comment by Jay8181** [comment]
I would like to take this issue : ).
